src/test/java/uk/gov/cshr/civilservant/service/OrganisationalUnitServiceTest.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.cshr.civilservant.service;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import uk.gov.cshr.civilservant.domain.OrganisationalUnit;
+import uk.gov.cshr.civilservant.repository.OrganisationalUnitRepository;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.hasProperty;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+public class OrganisationalUnitServiceTest {
+
+    private OrganisationalUnitService organisationalUnitService;
+
+    @Mock
+    private OrganisationalUnitRepository organisationalUnitRepository;
+
+    @Before
+    public void setup() {
+        initMocks(this);
+        organisationalUnitService = new OrganisationalUnitService(organisationalUnitRepository);
+    }
```

Comment:
The above would look a little cleaner as...

```
@RunWith(MockitoJUnitRunner.class)
public  class OrganisationalUnitServiceTest {
@Mock
private OrganisationalUnitRepository organisationalUnitRepository;

@InjectMocks
private OrganisationalUnitService organisationalUnitService;
```
You wouldn't then need the `setup()` method.